### DISCUSSION TITLE
Proposing a refreshed version of the digipres.org homepage

### DISCUSSION THREAD

**Discussion by anjackson** [digipres/digipres.github.io#52] [Announcements]
I've been spending some time experimenting with ideas for refreshing the digipres.org 'portal' homepage, and I'd like to get some wider feedback on what we should do.

The core of the idea is to replace the current page with a simpler interface that just sends folks to particular resources or services based on the kind of thing they are interested in. There are a few interrelated questions:

1. What a the 'big questions' that we should guide people to?
2. Should each of these correspond to a section of the Awesome List? Or does it make sense to link directly to some community resources?
3. What should it look like?

## Questions & Destinations

Currently, we have 'The Digital Preservation Awesome List' and the actions like:

- Find Preservation Tools
- Find File Formats
- Ask & Answer Digital Preservation Questions
- Digital Preservation Requirements & Solutions
- Get Started
- Save Digital Stuff Right Now

The first four go to particular services/sub-sites, although the Requirements & Solutions link is broken. The last two go to sections of the Awesome List.

I generally like the idea of every section being some kind of action, and I'm currently thinking that they should all link to a corresponding section of the Awesome List, for consistency. But this will add a step ahead of getting to some services, e.g. linking to a general _DigiPres Community_ section before getting to https://qanda.digipres.org/, rather than going there directly.  Similarly, a new _Find Formats_ section would go to a _Find Formats_ part of the Awesome List before one of those entries then offers to take you to the format aggregator. I think this is probably a good idea overall, but I'd value feedback on this because I'll mean overhauling the Awesome List at the same time.

## What should it look like?

I've been experimenting with a 'big friendly cards' style interface, and coming up with some graphics based on the illustrations generated for the digital preservation by the SPRUCE project.  The DPC also uses these, so I've attempted to establish a distinct style that makes used of how detailed those illustrations are, combined with a much more vibrant range of colours.

It's under [this PR](https://github.com/digipres/digipres.github.io/pull/51) and is visible at: https://deploy-preview-51--digipres-org.netlify.app/

![dpo-refresh-screenshot](https://github.com/user-attachments/assets/29ab9c49-8663-42c4-b6f5-b83c47c42b87)

## What should the full list of Questions be? What about Topics/Areas?

I've tried to flesh out the list of Questions somewhat, and make them easier to manage by using [a structured list](https://github.com/anjackson/digipres.github.io/blob/2024-homepage-refresh/index.md?plain=1#L6-L70) which powers the front page view shown above.

Currently, there are no links (at least not from the portal) to:

- [Digital Preservation Requirements & Solutions](http://wiki.opf-labs.org/display/REQ/Digital+Preservation+and+Data+Curation+Requirements+and+Solutions) (which is currently down, as noted above, but where should it be linked in?)
- [Ask & Answer Digital Preservation Questions](http://qanda.digipres.org/) (add to  a 'Join Us' section?)
- [The File Formats Wiki](http://fileformats.archiveteam.org/) (Add to a 'Find Formats' section?)
- [Start 2 Preserve Skills Resource](https://digital-preservation-coalition.github.io/learning/), [DPC Online Learning](https://dpc.getlearnworlds.com/courses) (Add to the 'Get Started' section?)
- [All the useful information resources listed here](https://github.com/digipres/registries-of-practice-project/issues/2)

There's also an issue with particular areas of interest, and how best to handle those. e.g. links relating to emulation, or A/V work, or digital forensics, or media migration in general, or floppy disks in particular, or web archiving, or digitisation projects, or PDF, or environmental issues, or AI, etc.  These often come up as topics where folks want to pool some knowledge, but I'm not sure how best to integrate that kind of need with this site.  I think maybe through classes of Workflows?  Again, any ideas, especially ones that have been shown to work elsewhere, are very welcome!

**Comment by anjackson** [comment]
Just commenting to note that @thorsted has a list of places to look for files of various formats: https://github.com/thorsted/fileformat which is similar to the 'format corpus' part of the Awesome List. I guess we can at least link to it from the Awesome List!?

EDIT: similarly, https://github.com/pdf-association/pdf-corpora and the blog these two from, https://digipres.fr/archives/69 by @BertrandCaron

**Comment by torbjornbp** [comment]
I haven't got much to add, apart from the fact that I really like new big friendly cards layout!

**Comment by anjackson** [comment]
I've merged and deployed the 'big friendly card' layout, as it was an improvement over the original and heads in the right direction. However, I think it needs further iteration and input so I'm not calling this done.

**Comment by anjackson** [comment]
As the feedback I have received has been positive, I'm going to close this discussion and proceed with the current homepage for now. I'll open new discussions on specific features I've been experimenting with.
